index.js

```diff
@@ -552,7 +552,7 @@ function getRedirectUrl (url) {
 }
 
 module.exports = fp(fastifyStatic, {
-  fastify: '4.x',
+  fastify: '4.23.x',
```

Comment:
It is wrong, `4.23.x` means only `4.23` is the supported version.
So, whenever a minor bump on `fastify`, it will not be able to use.

```suggestion
  fastify: '~4.23.0',
```

Reply:
Isn't this the same as 4.23.x? Shouldn't it be ^4.23.0?